src/local_scheduler/local_scheduler_shared.h

```diff
@@ -8,21 +8,20 @@
 #include "utarray.h"
 #include "uthash.h"
 
+#include <unordered_map>
 #include <vector>
 
 /* These are needed to define the UT_arrays. */
 extern UT_icd task_ptr_icd;
 
 /** This struct is used to maintain a mapping from actor IDs to the ID of the
  *  local scheduler that is responsible for the actor. */
-typedef struct {
+struct ActorMapEntry {
```

Comment:
We can actually get rid of this and make the hashmap from ActorID to DBClientID.

Storing the key in the struct was only neccessary for uthash, in the STL the key can always be gotten separately.